src/components/layout-item/layout-item.tsx

```diff
@@ -0,0 +1,127 @@
+import { Component, h, Host, Prop, State, VNode } from "@stencil/core";
+import { slotChangeHasAssignedElement } from "../../utils/dom";
+import { CSS, SLOTS } from "./resources";
+import { Layout } from "../interfaces";
+
+/**
+ * @slot - A slot for adding content.
+ * @slot actions-start - A slot for adding actionable `calcite-action` elements before the content of the component.
+ * @slot content-start - A slot for adding non-actionable elements before content of the component.
+ * @slot content-end - A slot for adding non-actionable elements after content of the component.
+ * @slot actions-end - A slot for adding actionable `calcite-action` elements after the content of the component.
+ */
+@Component({
+  tag: "calcite-layout-item",
```

Comment:
Is this going to be public or internal component? The “-item” portion of the name makes me think there would be a parent of some kind - layout / layout-group.

Reply:
‘calcite-content-row’ / ‘layout-row’ (if we don’t expect it to have a “column” layout) ?

Reply:
Good question. We can always just start with row layout and not have column until the need arises.